README.md

```diff
@@ -60,6 +60,41 @@ Experimenter is a platform for managing experiments in [Mozilla Firefox](https:/
 
 Done!
 
+### Semi Dockerized Setup
+
+[osx catalina, reinstall command line tools](https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d)
+
+1.  Pre reqs (macOs instructions)
+
+        brew install postgresql llvm openssl yarn
+
+        echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.bash_profile
+        export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
+
+2.  Install dependencies
+
+        poetry install
+
+        yarn install
```

Comment:
It's probably worth noting somewhere that Node ^14.0.0 is required. Had to restart at this step when I realized v12.18.2 wasn't going to cut it 😩

Reply:
oooh, will add the note